demos/3_Multichannel/03_Hyperspectral_reconstruction.ipynb

```diff
@@ -997,7 +975,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "recons = [FBP_recon_4D.get_slice(channel=20, vertical=35), PDHG_TV_100_Sp.get_slice(channel=20, horizontal_y=35),\\\n",
+    "recons = [FBP_recon_4D.get_slice(channel=20, vertical=35), PDHG_TV_100_Sp.get_slice(channel=20, horizontal_y=35),\n",
```

Comment:
Here we are comparing two different axes - vertical and horizontal_y with different recon methods. We should be comparing the same axis. I would suggest going with vertical?

Reply:
changing this is out of scope of this PR. Please open a separate issue/PR.

Reply:
done in this pr